lib/rules/jsx-no-target-blank.js

```diff
@@ -74,6 +74,29 @@ function getStringFromValue(value) {
       if (value.expression.type === 'TemplateLiteral') {
         return value.expression.quasis[0].value.cooked;
       }
+      const expr = value.expression;
+      if (expr.type === 'ConditionalExpression') {
+        if (
+          typeof expr.consequent.value === 'string'
+          && (
+            (expr.consequent.value && expr.consequent.value.toLowerCase() === 'noreferrer')
+            || (expr.consequent.value && expr.consequent.value.toLowerCase() === 'noopener noreferrer')
+            || (expr.consequent.value && expr.consequent.value.toLowerCase() === 'noreferrer noopener')
+          )
```

Comment:
seems worth putting this logic into a helper function, so it can be used in both places?